docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -754,6 +807,7 @@ Specific observer outputs and their stratifications may vary by model run as nee
     - 10
     - 100,000
     - Baseline
+    - ``mnt/team/simulation_science/pub/models/vivarium_gates_mncnh/results/azithromycin``
```

Comment:
For the models that have not been run yet, would we like to start using numbering convention for filepaths rather than names?

Reply:
YES!!!